contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -1901,3 +1901,52 @@ char
 		appendStringInfo(&query, "\"%s\".\"%s\"", schema_name, index_name);
 	return query.data;
 }
+
+/*
+ * Helper function to execute ALTER ROLE command using
+ * ProcessUtility(). Caller should make sure their
+ * inputs are sanitized to prevent unexpected behaviour.
+ */
+void
+exec_alter_role_cmd(char *query_str, RoleSpec *role)
```

Comment:
There is pre-existing function `exec_utility_cmd_helper` to execute the utility commands, I think we should use that instead with a little modification that it can take `Node*` as an argument. ie.
```
exec_utility_cmd_helper(char *query_str, Node *obj)
```
And then we can have switch case on Node type to update the generated parsetree depending upon the its type.

Reply:
Fyi the current `exec_utility_cmd_helper` has SQL injection risk for directly executing query string with statement number check (which is insufficient). We should always parse dummy query first and then update the Node with real value. 
But it is a good idea to integrate all query parsing/update into one function, as Rishabh mentioned.
There was a slack thread about this a month ago. I would suggest leave this change to a separate PR, but will be great to file a Jira for this.

Reply:
Ack. I will create a separate JIRA for this. Because that would need change at multiple places wherever exec_utility_cmd_helper is being used.

Reply:
https://jira.rds.a2z.com/browse/BABEL-4649 It can be tracked here.